charts/kyverno/values.yaml

```diff
@@ -11,6 +11,9 @@ global:
     # -- (string) Global value that allows to set a single image registry across all deployments.
     # When set, it will override any values set under `.image.registry` across the chart.
     registry: ~
+    # -- (list) Global list of Image pull secrets
+    # When set, it will override any values set under `imagePullSecrets` under different components across the chart.
+    imagePullSecrets: []
```

Comment:
This should be at the same level with `image`.

Reply:
@realshuting is this resolved now ?